src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/request/CreatedBy.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.companieshouse.api.strikeoffobjections.model.request;
+
+public class CreatedBy {
+    private String id;
+    private String forename;
```

Comment:
API has been updated. forename, surname were nice to have's and have been taken out. If we need names we can cross reference the user collection

Reply:
Names are gone.